wrlinux/wrlinux.go

```diff
@@ -0,0 +1,280 @@
+/*
+ * Copyright (c) 2022 Wind River Systems, Inc.
+ *
+ * The right to copy, distribute, modify, or otherwise make use
+ * of this software may be licensed only pursuant to the terms
+ * of an applicable Wind River license agreement.
+ */
+
+package wrlinux
+
+import (
+	"io"
+	"bufio"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/aquasecurity/vuln-list-update/git"
+	"github.com/araddon/dateparse"
+	"golang.org/x/xerrors"
+	"github.com/aquasecurity/vuln-list-update/utils"
+)
+
+const (
+	cveTrackerDir = "windriver-cve-tracker"
+	windriverDir  = "wrlinux"
+)
+
+var (
+	repoURLs = []string{
+		"https://distro.windriver.com/git/windriver-cve-tracker.git",
+	}
+	targets = []string{
+		"active",
+	}
+	statuses = []string{
+		"released",
+		"pending",
+		"not-affected",
+		"ignored",
+	}
+)
+
+type Vulnerability struct {
+	Candidate         string
+	PublicDate        time.Time
+	Description       string
+	References        []string
+	Notes             []string
+	Priority          string
+	Bugs              []string
+	Patches           map[Package]Statuses
+}
+
+type Package string
+
+type Release string
+
+type Statuses map[Release]Status
+
+type Status struct {
+	Status string
+	Note   string
+}
+
+func Update() error {
+	var err error
+	gc := git.Config{}
+	dir := filepath.Join(utils.CacheDir(), cveTrackerDir)
+	for _, url := range repoURLs {
+		_, err = gc.CloneOrPull(url, dir, "master", false)
```

Comment:
i think we can remove this folder after adding CVEs.
wdyt?

Reply:
absolutely!